app/controllers/searchjoy/searches_controller.rb

```diff
@@ -39,7 +39,8 @@ def recent
     protected
 
     def set_search_types
-      @search_types = Searchjoy::Search.uniq.pluck(:search_type).sort
+      @search_types = Searchjoy::Search.uniq.pluck(:search_type)
+      @search_types = @search_types.reject(&:blank?).sort + @search_types.select(&:blank?)
```

Comment:
When using searches across **multiple indices** (with searchkick) there is no `name` nor `klass` available, thus the `search_type` will be blank. This results in displaying in an error when trying to access the searchjoy statistics panel. (Can´t do `[nil, 'String'].sort`)